Modules/System/MicrosoftGraph/src/helper/MgOperationResponse.Codeunit.al

```diff
@@ -0,0 +1,97 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 9138 "Mg Operation Response"
+{
+    Access = Internal;
+
+    [NonDebuggable]
+    [TryFunction]
+    internal procedure GetResultAsText(var Result: Text);
+    var
+        ResultInStream: InStream;
+    begin
+        TempBlobContent.CreateInStream(ResultInStream);
+        ResultInStream.ReadText(Result);
+    end;
+
+    [NonDebuggable]
+    [TryFunction]
+    internal procedure GetResultAsStream(var ResultInStream: InStream)
+    begin
+        TempBlobContent.CreateInStream(ResultInStream);
+    end;
+
+    [NonDebuggable]
+    internal procedure SetHttpResponse(HttpResponseMessage: HttpResponseMessage)
+    var
+        ContentInStream: InStream;
+        ContentOutStream: OutStream;
+    begin
+        TempBlobContent.CreateOutStream(ContentOutStream);
+        HttpResponseMessage.Content().ReadAs(ContentInStream);
+        CopyStream(ContentOutStream, ContentInStream);
+        HttpHeaders := HttpResponseMessage.Headers();
+        MicrosoftGraphDiagnostics.SetParameters(HttpResponseMessage.IsSuccessStatusCode, HttpResponseMessage.HttpStatusCode, HttpResponseMessage.ReasonPhrase, GetRetryAfterHeaderValue(), GetErrorDescription());
+    end;
+
+    [NonDebuggable]
+    internal procedure SetHttpResponse(ResponseContent: Text; ResponseHttpHeaders: HttpHeaders; ResponseHttpStatusCode: Integer; ResponseIsSuccessStatusCode: Boolean; ResponseReasonPhrase: Text)
```

Comment:
internal access modifier is superfluous here, since the codeunit is already marked as internal.